src/arena.rs

```diff
@@ -240,6 +247,21 @@ impl<T> Arena<T> {
             marker: PhantomData,
         }
     }
+
+    #[allow(unused_variables)]
+    pub fn get_span(&self, handle: Handle<T>) -> Span {
+        #[cfg(feature = "span")] {
+            return self.span_info.get(handle.index()).map_or(Span::Unknown, |x| x.clone())
```

Comment:
Generally arena's access out of bounds panics (unless you use the "try_" method version). Any reason to not just panic here?